src/ui-kit/Button/Button.tsx

```diff
@@ -5,7 +5,20 @@ import { getRegularButtonClasses } from './getRegularButtonClasses';
 import { getDisabledButtonClasses } from './getDisabledButtonClasses';
 
 export const Button = JSX<ButtonWithIconProps>((props) => {
-  const { text, aboveText, appendLeft, appendRight, children, disabled, rounded, ...rest } = props;
+  const {
+    text,
+    aboveText,
+    appendLeft,
+    appendRight,
+    children,
+    disabled,
+    rounded,
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    iconRight, // its non valid HTML attribute. Must be excluded from "rest"
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    icon, // its non valid HTML attribute. Must be excluded from "rest"
+    ...rest
+  } = props;
```

Comment:
Может что-то вроде такого сделать? 

```suggestion
  // Pick non valid HTML attributes. Must be excluded from "rest"
  // eslint-disable-next-line @typescript-eslint/no-unused-vars
  const { asSVG, iconRight, icon, ...usedProps } = props;

  const { text, aboveText, appendLeft, appendRight, children, disabled, rounded, ...rest } =
    usedProps;
```

Reply:
Согласен. так красивее.